mabwiser/neighbors.py

```diff
@@ -21,10 +21,12 @@
 class _Neighbors(BaseMAB):
 
     def __init__(self, rng: _BaseRNG, arms: List[Arm], n_jobs: int, backend: Optional[str],
-                 lp: Union[_EpsilonGreedy, _Linear, _Popularity, _Random, _Softmax, _ThompsonSampling, _UCB1], metric: str):
+                 lp: Union[_EpsilonGreedy, _Linear, _Popularity, _Random, _Softmax, _ThompsonSampling, _UCB1],
+                 metric: str, no_nhood_prob_of_arm=None):
```

Comment:
For consistency, can you add type hints to this? `Optional[List[float]]` would probably work.